persister/whisper.go

```diff
@@ -77,14 +79,20 @@ type Whisper struct {
 			logicalSizeChanges  int64
 		}
 	}
-
+	prometheus whisperPrometheus
 	// blockThrottleNs        uint64 // sum ns counter
 	// blockQueueGetNs        uint64 // sum ns counter
 	// blockAvoidConcurrentNs uint64 // sum ns counter
 	// blockUpdateManyNs      uint64 // sum ns counter
 }
 
 // NewWhisper create instance of Whisper
+type whisperPrometheus struct {
+	enabled              bool
+	outOfOrderWritesLags prometheus.Histogram
```

Comment:
I think `outOfOrderWriteLags` is more digestable, lemme see if I can find an article on that

Reply:
https://english.stackexchange.com/a/258117